include/Util/Time.hpp

```diff
@@ -17,6 +17,8 @@ namespace Util {
  */
 class Time {
 public:
+    static void StartCounter() { s_Start = SDL_GetPerformanceCounter(); }
```

Comment:
Need docs for this method. Also, what the purpose of this method? `s_Start` already inited with `SDL_GetPerformanceCounter()`.